src/machine/aot/aot.x64.c

```diff
@@ -280,6 +280,53 @@ void aot_finalize(AotContext* context)
 int aot_link(AotContext* context, size_t *szp)
 {
   dasm_State** Dst = &context->d;
+  |->check_write:
```

Comment:
This procedure does not follow the calling convention, I think it's better to add some comment here that what parameters are expected stored in registers, and what is the return value and how to find it.

Reply:
Already added.